pkg/expr/expr.go

```diff
@@ -150,9 +151,14 @@ func newConstant(v string, vType config.ValueType) (*Constant, error) {
 			return nil, err
 		}
 	default: // string
```

Comment:
With the config.Duration fix, this is good for now.
Can we file a separate Issue for the remaining items?